doc/src/guides/index.rst

```diff
@@ -16,5 +16,6 @@ For a deeper and elaborate exploration of other SymPy topics, see the
    assumptions.rst
    booleans.rst
    custom-functions.md
+   physics-guides/index.rst
```

Comment:
```suggestion
   physics/index.rst
```

Reply:
the url already contains "guides", so you can just use "physics".

Reply:
Okay. I have updated this in the latest commit.